core/util/http.py

```diff
@@ -159,6 +159,10 @@ class RequestTimedOut(RequestNetworkException, requests.exceptions.Timeout):
 class HTTP:
     """A helper for the `requests` module."""
 
+    # In case an app version is not present, we can use this version as a fallback
+    # for all outgoing http requests without a custom user-agent
+    DEFAULT_USER_AGENT_VERSION = "1.x.x"
+
```

Comment:
Purely a personal preference thing here, but when I have to expose faked version numbers like this, I generally use `0.0.0` as the components are still parseable as numbers and `0.0.0` is unlikely to have ever been a real release.

Reply:
This is to retain current behavior. See `core/config.py#L190` above.